src/main/cpp/chassis/swerve/driveStates/RobotDrive.cpp

```diff
@@ -63,9 +63,12 @@ std::array<frc::SwerveModuleState, 4> RobotDrive::UpdateSwerveModuleStates(Chass
 
     wpi::array<frc::SwerveModuleState, 4> states = m_kinematics.ToSwerveModuleStates(chassisMovement.chassisSpeeds, chassisMovement.centerOfRotationOffset + m_centerOfRotation);
 
-    m_kinematics.DesaturateWheelSpeeds(*states, m_maxspeed);
+    m_kinematics.DesaturateWheelSpeeds(&states, m_maxspeed);
 
```

Comment:
Do we need this extra copying?  Can't we just return states?

Reply:
It doesn't look like we need the individual wheel states, I'll remove these